### ISSUE TITLE
Add the terminology DeviceType to study target? 

### ISSUE THREAD

**Issue by ingrre** [HumanBrainProject/openMINDS_controlledTerms#532]
In this dataset (https://search.kg.ebrains.eu/live/4282b8f8-0d34-47a0-b031-a449459e48c1) we share a locarejson file which specifies the 3d atlas position of the electrode used in the original experiment (more specifically, a 16-contact laminar electrode (Neuronexus A1x16–
100mm-100-43) with a 100 μm spacing between contacts). 'Electrode' would thus be a fitting study target.

**Comment by lzehl** [comment]
@apdavison @Peyman-N we currently have in deviceType:
+ CT scanner
+ MRI scanner
+ electronic amplifier
+ microscope # I think this is too generic and should be "light microscope"
+ microtome
+ vibrating microtome

For the request by @ingrre we could add:
+ linear multielectrode array

We should further think about what we want as other electrode / multielectrode arrray types:
+ wire electrode # for simple wires
+ suction electrode # relatively large; for nerve recordings
+ sharp microelectrode
+ metal microelectrode
+ pipette
???

**Comment by lzehl** [comment]
@apdavison @Peyman-N we should discuss this and make action points for resolving this issue

**Comment by Peyman-N** [comment]
Hi everyone
I just wanted to reopen this issue and bring it up in the next meeting, as @Alixbonard and @ree-gupta need it. 
Besides the ones suggested by @lzehl, I suggest adding following ones too, like:

1- Genral types

- Monopolar electrodes
- Bipolar electrodes
- Tetrodes
- Hexatrodes ?
- Octrodes ? 
- Multielectrode arrays ? (for ten or more)
- Silicon-based multichannel probes (e.g., Neuropixels, Michigan probes)
- Flexible polymer electrodes

2- Intracellular Recording
- Sharp glass microelectrodes
- Patch pipettes (for whole-cell patch clamp, cell-attached, etc.)

3- Field Potential Recording

- Glass micropipettes
- Metal wire electrodes (e.g., twisted pair stainless steel)

4- For EEG/EMG:

- Cup electrodes 
- Disk electrodes
- Dry electrodes
- Gel-based electrodes
- Needle electrodes
- Surface electrodes
- Hook electrodes (for nerve recordings in animals)
- Fine-wire electrodes

Some electrodes can belong to more than one category, so we should refine this list, but it can serve as a good starting point. For example, should we have `Patch pipettes` and `Glass micropipettes` or just one `Pipette`, I am leaning more toward one pipette. 
Alternatively, I would consider an alectrode array containing more than two electrodes as `Multielectrode arrays`. However, many sources consider them to be multielectrode arrays only after 10 electrodes. Should we also include arrays with 4, 6, and 8 electrodes?

What are your thoughts @apdavison @lzehl @UlrikeS91 
Do you think we should pin Tom in this discussion as well?

| Name                                | Definition                                                                                   | Description                                                                                                 | Synonym                         |
|-------------------------------------|----------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|----------------------------------|
| Monopolar electrodes                | Electrodes with a single active recording site referenced to a distant ground.              | Used in various neural and muscle recordings; suitable for isolating signals from a localized region.      | single-ended electrode          |
| Bipolar electrodes                  | Electrodes with two closely spaced contacts to measure the potential difference between them.| Common in stimulation and recording to reduce common-mode noise.                                           | differential electrode          |
| Tetrodes                            | A set of four closely spaced wires used to detect signals from multiple nearby neurons.     | Improves discrimination of multiple neurons based on signal differences across the wires.                  | quad-wire electrode             |
| Hexatrodes                          | Electrode bundle with six channels for recording or stimulation.                            | Used in neuroscience for denser spatial sampling than tetrodes.                                            | six-wire electrode              |
| Octrodes                            | Electrode assembly with eight channels.                                                     | Enables high-resolution recordings from local neural populations.                                          | eight-wire electrode            |
| Multielectrode arrays               | Array containing multiple electrode sites for simultaneous recording.                        | Often consists of 10 or more electrodes for population-level recordings.                                   | --                             |
| Silicon-based multichannel probes   | Microfabricated probes with multiple electrode sites on a silicon substrate.                | Examples include Neuropixels and Michigan probes, enabling high-density neural recordings.                 | silicon neural probes           |
| Flexible polymer electrodes         | Electrodes made of biocompatible flexible polymers.                                          | Adaptable to tissue surfaces, minimizing damage and improving chronic implant compatibility.               | flex-electrodes                 |
| Sharp glass microelectrodes         | Pulled glass pipettes with fine tips for intracellular recording.                           | Used for sharp electrode intracellular recordings, particularly in invertebrate systems.                   | sharp electrodes                |
| Sharp microelectrode                | A microelectrode with a sharp tip for penetrating cell membranes.                           | Common in intracellular recording for measuring membrane potentials.                                       | intracellular microelectrode    |
| Metal microelectrode               | Micron-scale metal electrode used for single-unit recordings.                              | Provides high spatial resolution for extracellular recordings.                                              | metallic microelectrode         |
| Patch pipettes                      | Glass pipettes used in patch clamp recordings.                                               | Used for whole-cell, inside-out, outside-out, and cell-attached patch clamp techniques.                    | patch clamp pipettes            |
| Pipette                             | Glass capillary tube pulled to a fine tip for electrophysiological use.                     | Can be used for intracellular recording, patch clamping, or substance injection.                           | micropipette                    |
| Glass micropipettes                 | Pulled glass pipettes used for intracellular or field potential recordings.                 | Can be filled with electrolytes and used for both current injection and recording.                         | glass electrodes                |
| Metal wire electrodes               | Simple metal wires used for extracellular field recordings.                                 | Often made of stainless steel or nichrome; twisted pairs reduce noise.                                     | wire electrodes                 |
| Wire electrode                      | Conductive wire used for bioelectric signal recording or stimulation.                        | Flexible and easy to implant in a variety of electrophysiological setups.                                  | single-wire electrode           |
| Cup electrodes                      | Small cup-shaped electrodes often used in EEG.                                               | Typically placed on the scalp with conductive gel to record brain activity.                                | EEG cup electrodes              |
| Disk electrodes                     | Flat disc-shaped electrodes.                                                                | Used for surface recordings, often with adhesive or gel.                                                   | flat electrodes                 |
| Dry electrodes                      | Electrodes that do not require gel or fluid for signal acquisition.                         | Used in wearable and mobile EEG systems for convenience and hygiene.                                       | gel-free electrodes             |
| Gel-based electrodes                | Electrodes that use conductive gel to improve skin contact.                                 | Common in EEG, ECG, and EMG for reducing impedance.                                                         | wet electrodes                  |
| Needle electrodes                   | Small metal needles inserted into tissue.                                                   | Used in EMG, peripheral nerve stimulation, or intraoperative monitoring.                                    | penetrating electrodes          |
| Surface electrodes                  | Electrodes placed on the surface of the skin.                                               | Used in non-invasive techniques like EEG, ECG, EMG.                                                         | skin electrodes                 |
| Hook electrodes                     | Wire hooks placed around nerves for recording or stimulation.                               | Used in animal studies for stable nerve contact.                                                            | nerve hook electrodes           |
| Fine-wire electrodes                | Thin wire electrodes inserted into muscles or tissues.                                      | Used in EMG for fine muscle group recordings.                                                               | intramuscular wire electrodes   |
| Suction electrode                   | Electrode that applies suction to hold a small structure in place.                          | Commonly used in invertebrate electrophysiology, such as for recording from nerves or ganglia.             | suction pipette electrode       |

**Comment by Alixbonard** [comment]
Most urgent to continue the in-depth curation: 

| Name | Definition | Description | Synonym |
|--------|--------|--------|--------|
|monopolar electrodes|Electrodes with a single active recording site referenced to a distant ground.|Used in various neural and muscle recordings; suitable for isolating signals from a localized region.|single-ended electrode|
| tetrodes | A set of four closely spaced wires used to detect signals from multiple nearby neurons. | Improves discrimination of multiple neurons based on signal differences across the wires. | quad-wire electrode|
| multielectrode arrays | Array containing multiple electrode sites for simultaneous recording. | Often consists of 10 or more electrodes for population-level recordings. | - |
| silicon-based multichannel probes | Microfabricated probes with multiple electrode sites on a silicon substrate. | Examples include Neuropixels and Michigan probes, enabling high-density neural recordings. | silicon neural probes |
| surface electrodes | Electrodes placed on the surface of the skin. | Used in non-invasive techniques like EEG, ECG, EMG. | skin electrodes | 

Question: 

Would it be redundant to include "bipolar electrode"?

**Comment by ree-gupta** [comment]
deviceType instances I need for NHP-EBRAINS datasets (based on suggestions already made): 

- multielectrodeArray,
-  siliconBasedMultichannelProbe, 
- fineWireElectrodes, 
- surfaceElectrodes,
-  gelBasedElectrode, 
- flexiblePolymerElectrodes 

further suggestions: 
- eyeTrackingSystem, 
- infraRedCamera, 
- 3DmotionCaptureSystem

**Comment by Alixbonard** [comment]
Based on the 08 July 2025 ontolongy meeting, the following may be added:
- single electrode
- 1D multi electrode array
- 2D multi electrode array
- 3D multi electrode array

Please feel free to correct

**Comment by ree-gupta** [comment]
I would also like to request terms used for devices recording eye movements, EMG, motion and pose estimation (hand and body) for my NHP-EBRAINS datasets. Or are there suggestions for another generic term be used for these devices meanwhile?

**Comment by lzehl** [comment]
This issue got to big and is not about the original question any more. Let's add the category studyTarget to DeviceType. 

For updating DeviceType I opened a new issue: https://github.com/openMetadataInitiative/openMINDS_instances/issues/446

**Comment by Raphael-Gazzotti** [comment]
Closed with #533.
